.golangci.yml

```diff
@@ -66,7 +66,7 @@ linters:
   enable:
     - bodyclose
     - deadcode
-    - depguard
+    #- depguard
```

Comment:
I'm already cleaning it up at #776, but I guess its fine. I'll have to rebase it anyway.